azkaban-web-server/src/test/java/azkaban/webapp/AzkabanWebServerTest.java

```diff
@@ -111,6 +113,11 @@ public void testInjection() throws Exception {
     executorLoader.updateExecutor(executor);
 
     assertNotNull(injector.getInstance(AzkabanWebServer.class));
+
+    //Test if triggermanager is singletonly guiced. If not, the below test will fail.
+    final TriggerManager triggerManager1 = requireNonNull(injector.getInstance(TriggerManager.class));
```

Comment:
requireNonNull seems odd. Why not assert the instance is not null using junit assert or assertj?